client/components/data/query-site-selected-editor/index.jsx

```diff
@@ -1,41 +1,23 @@
-/** @format */
 /**
  * External dependencies
  */
-import { Component } from 'react';
 import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
 
 /**
  * Internal dependencies
  */
 import { requestSelectedEditor } from 'state/selected-editor/actions';
 
-export class QuerySiteSelectedEditor extends Component {
-	static propTypes = {
-		siteId: PropTypes.number,
-	};
+export default function QuerySiteDomains( { siteId } ) {
```

Comment:
Again, why `QuerySiteDomains` instead of `QuerySiteSelectedEditor`? Looks like a copy and paste gone wrong 🙂

Reply:
Indeed 😅